environment/README.md

```diff
@@ -42,46 +42,78 @@ supported** now. If you want to run on Windows, use the Docker image instead.
 1. Create a conda environment and install main packages:
 
     ```bash
+   conda config --set channel_priority strict
    conda env create --name phenoplier --file environment.yml
    conda run -n phenoplier --no-capture-output bash scripts/install_other_packages.sh
     ```
-If the `conda env create` command fails or if you find package errors later, try
-to set the channel priority in your conda installation to "strict" with
-`conda config --set channel_priority strict`.
 
 1. Download the data:
 
    ```bash
    conda run -n phenoplier --no-capture-output python scripts/setup_data.py
    ```
 
-This will download ~70 GB of data needed to run the analyses.
+This will download ~130 GB of data and software needed to run the analyses.
 
 
 # Developer usage
 
 These steps are only for PhenoPLIER developers.
+All steps are run from the root directory (not within `environment/`).
 
-1. Modify `scripts/environment_base.yml` accordingly (if needed).
-1. Run:
+It is a good idea to try to build the environment locally first and, when all issues have been solved, then create the Docker image.
+A usual problem is to use a too recent Python version that produces several conflicts in conda.
+In that case, a previous Python version should be used instead.
```

Comment:
Could you include an approximate version number that you think will work at?

Reply:
Hm, maybe the instructions here are not very clear. That is already provided in the [environment.yml](https://github.com/greenelab/phenoplier/blob/main/environment/environment.yml) file. The instructions here are for a developer (me, basically, so I remember how to proceed).

I will try to improve this part.